_docker/drupal/drupal-filesystem/web/themes/custom/rhdp/templates/events/node--events--tile.html.twig

```diff
@@ -0,0 +1,23 @@
+{%
+  set classes = [
+    'node',
+    'node--type-' ~ node.bundle|clean_class,
+    node.isPromoted() ? 'node--promoted',
+    node.isSticky() ? 'node--sticky',
+    not node.isPublished() ? 'node--unpublished',
+    view_mode ? 'node--view-mode-' ~ view_mode|clean_class,
+  ]
+%}
+{{ attach_library('classy/node') }}
+<article{{ attributes.addClass(classes) }}>
+  <a href="{{ url }}">
+    <div class="tile-image">
+      <div class="image-table">
+        <div class="image-cell">
+          <img src="{{ content.field_tile_thumbnail }}"/>
+        </div>
+      </div>
+    </div>
+    <h3 class="tile-title">{{ event_title }}</h3>
+  </a>
+</article>
```

Comment:
Using CSS grid, I don't think we need as deeply nested a structure

Reply:
I agree, this does work fine with just the image and all div's stripped out. 

The reason I didn't do that here is because this is pretty much a direct copy from `.../video-resource/node--video-resource--tile.html.twig` and I wasn't sure where else you might want to use this so I figured it was safer to keep with the existing structure in case some other view might rely on it.

If that's not the case and you just want me to remove it, that's fine too.

Reply:
I think we'll want to go forward with less markup. We'll go back eventually and clean-up the other stuff.

Reply:
Cool, that's pushed. in 25ec393